components/LineChart.tsx

```diff
@@ -0,0 +1,131 @@
+import { Container, ScrollView } from "native-base"
+import React from "react"
+import { Dimensions } from "react-native";
+import {
+  AbstractChart,
+    LineChart
+  } from "react-native-chart-kit";
+import { G, Svg, Text } from "react-native-svg";
+
+
+const LineGraph = ({graphData}) => {
+
+    const options = {
+        responsive: true,
+        plugins: {
+          legend: {
+            position: 'none',
+          },
+        },
+    };
+
+    const labels = [];
+    const data = [];
+
+    if (graphData !== undefined && graphData.length > 0) {
+        for (let pos = graphData.length-1 ; pos >= 0; pos--) {
+          labels.push(graphData[pos].month);
+          data.push(graphData[pos].points);
+        }
+    }
+    
+    const chartConfig = {
+        backgroundColor: "#FFFFFF",
```

Comment:
nit: we can save these variables in constants as we did with the colors

Reply:
You're right. I'll move a lot of variables here to constants in another PR next sprint